ocs_ci/ocs/resources/storage_cluster.py

```
@@ -1546,79 +1550,138 @@ def verify_storage_device_class(
+            f"{crush_device_class_msg} but it should be" f" set to {device_class}"
+        )
 
     sc_device_classes = deviceset_name_per_deviceclass.values()
-    # get deviceClasses for overall storage
     device_classes = cephcluster_data["items"][0]["status"]["storage"]["deviceClasses"]
     log.debug(f"deviceClasses are {device_classes}")
     for each_device_class in device_classes:
         device_class_name = each_device_class["name"]
+        if device_class_name not in expected_device_classes:
+            log.info(
+                "Skipping operator-generated " "device class '%s'",
```

Ruff (0.16.1)

[warning] Implicitly concatenated string literals on one line

Combine string literals

(ISC001)